src/reducers/index.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
the `d.ts` file is converted and the `reducers/index.js` file is deleted here

Reply:
Actual transformations with the index file can be viewed in the second commit, with the name "[Moved types from d.ts files](https://github.com/keplergl/kepler.gl/pull/1722/commits/86d7c441e9095287080ab0cc6dfbe5c9f050e990)"

Reply:
I changed the composition of the PR, now the changes to the index file are displayed correctly.